docs/docs/vulnerability/detection/data-source.md

```diff
@@ -1,3 +1,12 @@
+# Important
+Trivy **only** receives advisories from the sources listed in the following tables.
+
+Also, for packages installed from package managers (`dpkg`, `yum`, `apk` and etc.), Trivy gets advisories from the appropriate **OS database**.
+
+For example: for a python package installed from `yum` (Amazon linux), Trivy will only get advisories from the `amazon database`. But for a  python package installed from another source (e.g. `pip`), Trivy will get advisories from the `GitLab` and `GitHub` databases.
+
+This is to avoid false positives when the vendor's OS fixes the vulnerability before the package owner fixes it.
```

Comment:
```suggestion
This advisory selection is essential to avoid getting false positives because OS vendors usually backport upstream fixes, and the fixed version can be different from the upstream fixed version.
```